### PR TITLE
nodePackages_8_x.re-natal: init at 0.8.2

### PR DESCRIPTION
**Pull request by peterromfeldhk**
nodePackages_8_x.re-natal: init at 0.8.2
nodePackages_8_x.react-native-cli: init at 2.0.1

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [ ] NixOS
   - [x] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/development/node-packages/node-packages-v8.json [hunk 1]
```diff
@@ -9,6 +9,8 @@
 , "node-gyp-build"
 , "node-pre-gyp"
 , "pnpm"
+, "react-native-cli"
+, "re-natal"
```
Comment (Mic92) [inline pkgs/development/node-packages/node-packages-v8.json]:
```
$ find /nix/store/ikqc7gc5ipq9jr5lharfsqpix5754003-node-re-natal-0.8.2/bin
```

contains no files, how is it supposed to be used?

**Comment by Mic92** [comment]
@GrahamcOfBorg build nodePackages_8_x.re-natal

**Comment by adisbladis** [comment]
There are a few [dependencies](https://github.com/drapanjanas/re-natal#dependencies) of  `re-natal` that should be wrapped/passed as buildInputs.

https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/node-packages/default-v8.nix

**Comment by Mic92** [comment]
Outdated. Please open a new pull request if you are still interested.

**Pull request closed without merging**